ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java

```diff
@@ -36,8 +34,6 @@ public void AddDiagnosisToVisitNoteTest() throws Exception {
         assertEquals("Pneumonia", visitNotePage.primaryDiagnosis());
         assertEquals("Bleeding", visitNotePage.secondaryDiagnosis());
         visitNotePage.save();
-        assertNotNull(patientDashboardPage.visitLink());
```

Comment:
I was previously working on this test, which got merged after passing through the CI. However, it failed a few days later on the [CI](https://ci.openmrs.org/browse/REFAPP-OMODDISTRO-4441/log) at that line, even though it was working fine on my local machine.
Previously, what visitLink() does is to detect a success toast pop up after saving the note. Back then, the test failed as it could not detect it for some reason (maybe there was no popup after saving it?), and caused a timeout. So what the change does is to detect another web element in the page instead; whether the page is returned to successfully, is the main objective.

On a side note, it seems like the log for UI tests is not displaying correctly. It used to display the part with the full error stack, but now it's only displaying the bottom part of it.